goodprice/demo/src/test/java/com/example/test/pages/KBBMOOCProject/carForSalePriceRatingComponent.java

```diff
@@ -0,0 +1,70 @@
+package com.example.test.pages.KBBMOOCProject;
+
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class carForSalePriceRatingComponent extends KbbBasePage {
+
+    public carForSalePriceRatingComponent(WebDriver driver) {
+        super(driver);
+        //TODO Auto-generated constructor stub
+    }
+    String titlePage ="Cars for Sale (Test Drive at Home) - Kelley Blue Book";
+    public String getTitle=driver.getTitle();
+    //*Verify Title Page */
+    public boolean checkTitlePage()
+    {
+        System.out.println(getTitle);
+        if(getTitle.equals(titlePage))
+        {
+            return true;
+
+        }return false;
+    }
+    //*Click Filter Price Rating  */
+    public void clickFilterPriceRating () throws InterruptedException{
+        WebElement txtPriceRating = keyword.findWebElementByXpath("//span[@data-cmp='filterTitle']/span[text()='Price Rating']");
+        keyword.scrollAndWaitToClick(txtPriceRating);
+    }
+    //*check Price Rating Filter Should Be Extended After Click */
+    public boolean chekPriceRatingFilterAfterClick() {
+        WebElement priceRatingFilterAfterExtending = keyword.findWebElementByXpath("//span[@data-cmp='filterTitle']/span[text()='Price Rating']/parent::span/parent::div/following-sibling::div");

```

Comment:
should write the web element locator in separate line then call into a method/function